src/worker/ingestion/process-event.ts

```diff
@@ -331,17 +331,29 @@ export class EventsProcessor {
         }
 
         if (oldPerson && newPerson && oldPerson.id !== newPerson.id) {
-            await this.mergePeople([newPerson, distinctId], [oldPerson, previousDistinctId], totalMergeAttempts)
+            await this.mergePeople({
+                totalMergeAttempts,
+                mergeInto: newPerson,
+                mergeIntoDistinctId: distinctId,
+                otherPerson: oldPerson,
+                otherPersonDistinctId: previousDistinctId,
+            })
         }
     }
 
-    public async mergePeople(
-        mergeIntoPayload: [Person, string],
-        otherPersonPayload: [Person, string],
-        totalMergeAttempts = 0
-    ): Promise<void> {
-        const [mergeInto, mergeIntoDistinctId] = mergeIntoPayload
-        const [otherPerson, otherPersonDistinctId] = otherPersonPayload
+    public async mergePeople({
```

Comment:
Q: Why struct over just 5 params?

Reply:
Honestly, I find it cleaner here. Had quite a bit of back and forth in my head trying to figure out what ID and person mapped to what ("oldPerson", "newPerson", "mergeInto", "otherPerson", "previousDistinctId", "distinctId"), so I like the explicitness in the call